tools/downloader/list_topologies.yml

```diff
@@ -1,3480 +0,0 @@
-# Copyright (c) 2018 Intel Corporation
```

Comment:
should not we put Copyright notice everywhere then? @yssemaev , your opinion?

Reply:
I'd be fine with doing that.

Reply:
Well, I just went ahead and added the full banner to each config file.